contents/blog/data-warehouse-at-posthog.mdx

```diff
@@ -0,0 +1,353 @@
+---
+title: How we use the data warehouse at PostHog
+date: 2025-07-14
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/warehouse_blog_633055c72f.png
```

Comment:
It's driving me mad, but I'm sure we have a hog illustration that has a bunch of SQL on a chalkboard or something. If we can find it, it might make a better lead image than this one?